src/styles/index.scss

```diff
@@ -1,3 +1,116 @@
-body {
+$day-size: 100px;
+$gap-size: 1px;
+$border-size: 1px;
+
+.page__body,
+.calendar__day {
+  box-sizing: border-box;
+}
+
+.page__body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+  width: (7 * $day-size) + ($gap-size * 6);
```

Comment:
There is an issue with the calculation of the width. In Sass, you need to use interpolation to perform arithmetic operations within a property value. You should wrap the entire calculation in `#{}` to ensure it is evaluated correctly.